manifests/firewall.pp

```diff
@@ -11,9 +11,9 @@
   if $monit::httpd_bool and $monit::manage_firewall_bool {
     if defined('::firewall') {
       firewall { "${monit::httpd_port} allow Monit inbound traffic":
-        action => 'accept',
-        dport  => $monit::httpd_port,
-        proto  => 'tcp',
+        jump  => 'accept',
```

Comment:
I guess it was renamed in the firewall module some time ago, right?

Reply:
I see the answer in the commit history, nvm

Reply:
I have been searching for the Release of puppetlabs-firewall, that introduces the jump and drops the action parameter. I would have expected that this change would brake the CI somehow, but I doesnt. I will see if I find a good solution to this whole thing